src/main/java/uk/gov/hmcts/reform/refunds/controllers/RefundsController.java

```diff
@@ -78,8 +75,6 @@ public class RefundsController {
     @Autowired
     private RefundNotificationService refundNotificationService;
 
-    private static final Logger LOG = LoggerFactory.getLogger(RefundsServiceImpl.class);
```

Comment:
Logger is required for any class. At a bare minimum, the incoming request should be Logged.